src/test/java/org/kohsuke/github/GitHubStaticTest.java

```diff
@@ -0,0 +1,72 @@
+package org.kohsuke.github;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.text.SimpleDateFormat;
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
+import java.util.Date;
+import java.util.TimeZone;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.CoreMatchers.not;
+
+/**
+ * Unit test for {@link GitHub} static helpers.
+ *
+ * @author Liam Newman
+ */
+public class GitHubStaticTest extends Assert {
+
+    @Test
+    public void timeRoundTrip() throws Exception {
+        Instant instantNow = Instant.now();
+
+        Date instantSeconds = Date.from(instantNow.truncatedTo(ChronoUnit.SECONDS));
+        Date instantMillis = Date.from(instantNow.truncatedTo(ChronoUnit.MILLIS));
+
+        // TODO: other formats
+        String instantFormatSlash = formatDate(instantMillis, "yyyy/MM/dd HH:mm:ss ZZZZ");
+        String instantFormatDash = formatDate(instantMillis, "yyyy-MM-dd'T'HH:mm:ss'Z'");
+        String instantFormatMillis = formatDate(instantMillis, "yyyy-MM-dd'T'HH:mm:ss.S'Z'");
+        String instantSecondsFormatMillis = formatDate(instantSeconds, "yyyy-MM-dd'T'HH:mm:ss.S'Z'");
+        String instantBadFormat = formatDate(instantMillis, "yy-MM-dd'T'HH:mm'Z'");
+
+
```

Comment:
I would like to see a short comment above each test to explain what is being checked. e.g. // Millisecond values should be ignored.

Reply:
I think 
https://github.com/github-api/github-api/blob/master/src/test/java/org/kohsuke/github/GitHubStaticTest.java#L48
covers this.   Cool?

Reply:
Looks good, thanks.